build-system/tasks/helpers.js

```diff
@@ -92,6 +91,14 @@ const watchDebounceDelay = 1000;
  */
 const cache = new Map();
 
+/**
+ * Used to remove a file from the babel cache after it is modified.
+ * @param {string} filepath relative to the project root.
+ */
+function invalidateBabelCache(filepath) {
```

Comment:
Nit: Since so many of our build and test tasks use babel, WDYT about calling this something like `invalidateUnminifiedBabelCache` or `invalidateEsbuildBabelCache`?

Reply:
Done